tests/manage/z_cluster/test_remove_mon_from_cluster.py

```diff
@@ -55,6 +55,7 @@ def run_io_on_pool(pool_obj):
 
 @tier4
 @tier4c
+@pytest.mark.skip("Test case is disabled, as per requirement not to support this scenario")
```

Comment:
maybe better to really disable it so it will not affect the reporting as skipped?
We can do this by removing all tier decorators

Reply:
you're right. Will do that